### ISSUE TITLE
Handle Underline in Markdown

### ISSUE THREAD

**Issue by irmerk** [accordproject/markdown-editor#98]
# To Be Done Instead of OR After Issue #97

Since transitioning to [utilize the `markdown-transform`](https://github.com/accordproject/markdown-editor/releases/tag/v0.6.0), we will need to take a step back from supporting underline for the time being. Markdown doesn't natively support underline, so we need to rethink how we serialise this. 

We can then experiment with using `html` tags for underline.

**Comment by HTMLGhozt** [comment]
Since #97 has a PR at I'll start to take a look here. I'm assuming we're looking at `<u></u>` to underline.

**Comment by ahmedkrmn** [comment]
Since issue #97 is now closed, can I start working on this?

**Comment by irmerk** [comment]
Yes @ahmedkrmn! Keep a lookout for the changes @dselman and I are making with the [`markdown-transform`](https://github.com/accordproject/markdown-transform/releases), because there may be an update that could possibly affect how you approach this. Not sure if it actually will, just keep an eye out. We should be finished with that either tomorrow or early Monday (October 6/7).

**Comment by irmerk** [comment]
@ahmedkrmn what is the status of this? I'm opening it back up for the time being.
